tests/torch/sota_checkpoints_eval.json

```diff
@@ -250,7 +250,7 @@
                 "config": "examples/torch/classification/configs/binarization/resnet18_imagenet_binarization_xnor.json",
                 "reference": "resnet18_imagenet",
                 "target_ov": 61.82,
-                "target_pt": 61.74,
+                "target_pt": 61.88,
```

Comment:
Measured metric for PT is not stable in result of using CUDA. Better increase target diff
```
              "diff_target_pt_min": -0.3,
              "diff_target_pt_max": 0.3
```

Reply:
Agree, done